### ISSUE TITLE
Alias Edit Button appears wrong spot (was menu slider)

### ISSUE THREAD

**Issue by baconpaul** [surge-synthesizer/surge#6822]
Tab into the shape slider and arrow down/up to the aditive option. What happen, or doesn't I guess is the open editor button doesn't appear. Don't know if it's just a accessibility or also a visual thing but it's not reachable either with VO or shift+tabbing back

Reminder to self to check this

**Comment by mkruselj** [comment]
Visually it works fine it seems.

**Comment by baconpaul** [comment]
Yeah I just did a push to make it closer to what the menu does which helps a lot

**Comment by baconpaul** [comment]
Confirmed the change above improves the reset-slider-name behavior. Not sure why the edit button is so hard to find when going additive though; maybe need to change a tab order?

**Comment by mkruselj** [comment]
@baconpaul Can we close this one?

**Comment by baconpaul** [comment]
There are a couple of things that seem to work fine throughout Surge, except for in the additive editor which is part of alias. The first being the inability to hit right bracket to do a right-click (which is a cool feature BTW, not sure when that got added in but I discovered it by accident like a month ago). The second is that the button to invoke/dismiss the editor doesn't respond to the enter key.

I remember a pass was done to fix a lot of these types of enter key issues all over the place, but that must have been forgotten about.

**Comment by baconpaul** [comment]
Oh that’s a comment from bacross on discord
